reana/cli.py

```diff
@@ -0,0 +1,511 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2018 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# REANA is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# REANA; if not, write to the Free Software Foundation, Inc., 59 Temple Place,
+# Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+"""Helper scripts for REANA developers. Run `reana --help` for help."""
+
+import os
+import subprocess
+import sys
+
+import click
+
+SRCDIR = os.environ.get('REANA_SRCDIR',
+                        os.path.expanduser('~/private/project/reana/src'))
```

Comment:
Maybe `os.getcwd()` as more expectable default?

Reply:
I think the safest would be to exit with an error when the two environment variables are not set.  The `cwd` may be good for initial cloning, but not for later operations if people are in a different directory...